tests/rptest/tests/admin_uuid_operations_test.py

```diff
@@ -183,6 +184,7 @@ def wait_until_cluster_healthy(self, timeout_sec=30):
                 timeout_s=timeout_sec,
                 backoff_s=1))
 
+    @skip_fips_mode
```

Comment:
@oren @michael-redpanda please review the commit message of this commit and let me know what you think. I am planning to update the node uuid runbook to call out that using the license enforcement escape hatch is also likely needed in enterprise clusters when using the uuid override feature.

Reply:
Worthwhile writing a test for this specific scenario?  e.g. being able to set the license escape hatch when doing UUID override?  It sounds like something we'll need in the not-to-distant future with licenses.

Reply:
Good idea, I'll add this.